sechub-webui/src/main/java/com/mercedesbenz/sechub/webui/SecHubServerAccessService.java

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.webui;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SecHubServerAccessService {
+    @Value("${sechub.serverUrl}")
```

Comment:
In SecHub server and PDS we always use lower cased spring values.

Please change to lowercase